src/victory-util/prop-types.js

```diff
@@ -90,15 +91,14 @@ export default {
    * @param {Array} validators Validation functions.
    * @returns {Function} Combined validator function
    */
+  // TODO(RYAN): Check with Lauren.
   allOfType(validators) {
-    return makeChainable((props, propName, componentName, ...rest) => {
-      const error = validators.reduce((result, validator) => {
-        return result || validator(props, propName, componentName, ...rest);
-      }, undefined);
-      if (error) {
-        return error;
-      }
-    });
+    return makeChainable((props, propName, componentName, ...rest) =>
```

Comment:
@boygirl -- I'm going to remove my `TODO` comment above, but I'd like your specific review and approval of my refactor here, since it should be the same, but I want to double check.